app/src/main/java/com/safetyheads/akademiaandroida/network/YouTubeApi.kt

```diff
@@ -1,6 +1,11 @@
 package com.safetyheads.akademiaandroida.network
 
 object YouTubeApi {
-
-    private const val BASE_URL = "https://www.googleapis.com/youtube/v3/"
+    const val YOUTUBE_API_BASE_URL = "https://www.googleapis.com/youtube/v3/"
+    const val YOUTUBE_API_KEY = "AIzaSyBjR-lXo3VdEoCfdhW53d_3HqOzUnKdit0"
+    const val YOUTUBE_CHANNEL_ID = "UCE8z3lh2LxX7WDKjli3gleg"
```

Comment:
I think it's good practice to hide the api key, you can check last meeting in Thursday, msarelo said something about it. Check also this discussion: https://github.com/SafetyHeads/Akademia-Androida-2023/pull/75#discussion_r1158704785